README.md

```diff
@@ -19,6 +19,8 @@ To be able to run this app, you need to create a .env file with the following fi
 `REACT_APP_PUBLICATION_CHANNEL_API_URL=https://api.nsd.no/dbhapitjener/Tabeller/hentJSONTabellData`<br>
 `REACT_APP_ALMA_API_URL=https://api-eu.hosted.exlibrisgroup.com`<br>
 `REACT_APP_CRISTIN_API_URL=https://api.cristin.no/v2`<br>
+`REACT_APP_API_URL=https://api.dev.nva.aws.unit.no`<br>
+`REACT_APP_API_TOKEN_KEY=XXXXXXXXXX`<br>
```

Comment:
TODO: `REACT_APP_API_TOKEN_KEY: REACT_APP_API_TOKEN_KEY` etc må leggess inn iparameter-store for buildspec.yml